src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const VERY_SHORT_TERM = 2;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCAUNT = 50;
+  const OVER_SHORT_TERM = 6;
+  const PRE_SHORT_TERM = 3;
+  const SHORT_TERM_DISCAUNT = 20;
+  const basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCAUNT;
+  }
+
+  if (days <= VERY_SHORT_TERM) {
+    return PRICE_PER_DAY * VERY_SHORT_TERM;
+  }
+
+  if (days >= PRE_SHORT_TERM || days <= OVER_SHORT_TERM) {
+    return basePrice - SHORT_TERM_DISCAUNT;
```

Comment:
The condition 'days >= PRE_SHORT_TERM || days <= OVER_SHORT_TERM' is incorrect and redundant. You should only check if 'days' is greater than or equal to 3 and less than 7 to apply the short term discount. Also, '||' (OR) should be '&&' (AND) to correctly represent the range.